### PR TITLE
table component has been added; API enhancements

### PR DESCRIPTION
**Pull request by borysyuk**

### CODE REVIEW

File: src/components/chart/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import Api from '../../api'
+import Api from '../../api/index.js'
```
Comment (pwalsh) [inline src/components/chart/index.js]: You should not need that. Also you should not ever need or use `.js` extension in imports.

File: src/components/pie/index.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import {Api} from '../../api'
+import {Api} from '../../api/index.js'
```
Comment (pwalsh) [inline src/components/pie/index.js]: Convention is `{ Api }`, not `{Api}`

**Comment by pwalsh** [comment]
@borysyuk fix those style issues and then good to merge.

**Comment by borysyuk** [comment]
@pwalsh fixed!

**Pull request merged**